telemetry-core/src/main/java/com/newrelic/telemetry/events/json/EventBatchMarshaller.java

```diff
@@ -36,9 +40,44 @@ public String toJson(EventBatch batch) {
             .getTelemetry()
             .stream()
             .map(decorator)
-            .map(new EventToJson())
+            .map(EventBatchMarshaller::mapToJson)
```

Comment:
No reason to create a new `EventToJson` mapper instance each time this is invoked. Can refactor `EventToJson` to be a method whose signature matches `Function<String, String>` and use a method reference instead.